extension/chrome/settings/modules/add_key.ts

```diff
@@ -100,9 +98,13 @@ View.run(class AddKeyView extends View {
     try {
       const checked = await this.keyImportUi.checkPrv(this.acctEmail, String($('.input_private_key').val()), String($('.input_passphrase').val()));
       if (checked) {
-        await KeyStore.add(this.acctEmail, checked.encrypted); // resulting new_key checked above
-        const storageType: StorageType = ($('.input_passphrase_save').prop('checked') && !this.clientConfiguration.forbidStoringPassPhrase()) ? 'local' : 'session';
-        await PassphraseStore.set(storageType, this.acctEmail, { longid: KeyUtil.getPrimaryLongid(checked.encrypted) }, checked.passphrase);
+        await saveKeysAndPassPhrase(this.acctEmail, [checked.encrypted]); // resulting new_key checked above
```

Comment:
@tomholub `saveKeysAndPassPhrase` differs from plain `KeyStore.add` in a way that it saves the corresponding public keys in ContactStore for all the user's own email aliases.
Should we plan to use this helper function instead of `KeysTore.add` in all the remaining places in a separate task?

Reply:
Yes, we should consider doing that in a separate task, and also renaming that method. If it does that, it's not too appropriately named. Maybe `savePrvAndPubKeysAndPassPhrases`